<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShapeDock</class>
 <widget class="QDockWidget" name="ShapeDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="listButton">
        <property name="text">
         <string>List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="treeButton">
        <property name="text">
         <string>Tree</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="MyTreeView" name="shapeTree"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTreeView</class>
   <extends>QTreeView</extends>
   <header>mydockwidgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
